@here/harp-mapview/lib/MapView.ts

```diff
@@ -1624,9 +1624,22 @@ export class MapView extends THREE.EventDispatcher {
         pitchDeg: number = 0
     ): void {
         if (this.projection.type === ProjectionType.Planar) {
+            const distanceToGround = MapViewUtils.calculateDistanceToGroundFromZoomLevel(
```

Comment:
Please add comment what are you trying to achieve.
This doesn't look like it has anything to do with synchronous rendering,